packages/account/src/Components/financial-details/financial-details-partials.tsx

```diff
@@ -1,966 +1,121 @@
-    values,
-    handleChange,
-    handleBlur,
-    touched,
-    errors,
-    setFieldValue,
-    other_instruments_trading_frequency_enum,
-}: Partial<FormikProps<{ other_instruments_trading_frequency: string }>> & TOtherInstrumentsTradingFrequency) => (
-    <Field name='other_instruments_trading_frequency'>
-        {({ field }: FormikValues) => (
-            <React.Fragment>
-                <DesktopWrapper>
-                    <Dropdown
-                        placeholder={localize('Trading frequency in other financial instruments')}
-                        is_align_text_left
-                        name={field.name}
-                        list={other_instruments_trading_frequency_enum}
-                        value={values?.other_instruments_trading_frequency}
-                        onChange={handleChange}
-                        handleBlur={handleBlur}
-                        error={
-                            touched?.other_instruments_trading_frequency && errors?.other_instruments_trading_frequency
-                        }
-                        list_portal_id='modal_root'
-                        {...field}
-                        required
-                    />
-                </DesktopWrapper>
-                <MobileWrapper>
-                    <SelectNative
-                        placeholder={localize('Please select')}
-                        name={field.name}
-                        label={localize('Trading frequency in other financial instruments')}
-                        list_items={other_instruments_trading_frequency_enum}
-                        value={values?.other_instruments_trading_frequency}
-                        error={
-                            touched?.other_instruments_trading_frequency && errors?.other_instruments_trading_frequency
-                        }
-                        onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
-                            if (typeof handleChange === 'function') {
-                                handleChange(e);
-                            }
-                            if (typeof setFieldValue === 'function') {
-                                setFieldValue('other_instruments_trading_frequency', e.target.value, true);
-                            }
-                        }}
-                        {...field}
-                        required
-                    />
-                </MobileWrapper>
-            </React.Fragment>
-        )}
-    </Field>
-);
+import {
+    getAccountTurnoverList,
+    getEducationLevelList,
+    getEmploymentIndustryList,
+    getEstimatedWorthList,
+    getIncomeSourceList,
+    getNetIncomeList,
+    getOccupationList,
+    getSourceOfWealthList,
+} from 'Configs/financial-details-config';
+
+type TFinancialDetailsDropdownFieldProps = {
+    dropdown_list: Array<object>;
+    field_key: string;
+    placeholder?: string;
+    label: string;
+};
+
+const FinancialDetailsDropdownField = ({
```

Comment:
Should we move this component to the separate file?

Reply:
Not needed as that is the only component there. The other is a wrapper